packages/template-typescript-minimal/app/ssr.js

```diff
@@ -39,7 +39,6 @@ const {handler} = runtime.createHandler(options, (app) => {
 
     app.get('/favicon.ico', runtime.serveStaticFile('static/favicon.ico'))
 
-    app.get('/worker.js(.map)?', runtime.serveServiceWorker)
```

Comment:
This template does not register any service worker. Thus, we can remove this endpoint.